corehq/apps/reports/standard/cases/data_sources.py

```diff
@@ -176,7 +178,17 @@ def _get_username(self, user_id):
         return username
 
     def parse_date(self, date_string):
-        return string_to_utc_datetime(date_string)
+        try:
+            return string_to_utc_datetime(date_string)
+        except:
+            try:
```

Comment:
I'm hoping to figure out a way to do this that doesn't flood our email, for example by emailing only the 1st, 2nd, 4th, 8th, 16th, 32nd, etc. time that a particular "soft error" happens. Let me know if you have a better idea—but it'd be nice to be able to sprinkle these "soft asserts" into code as a way to put out feelers before a big refactor. That's one way I could have avoided a number of the problems from last night